### ISSUE TITLE
Insolation terms

### ISSUE THREAD

**Issue by kandersolar** [hackingmaterials/pv-terms#15]
The list doesn't include terms for insolation (integrated irradiance, Wh/m^2).  Seems like most or all of the terms in the "Irradiance" category could have insolation equivalents.

**Comment by cwhanse** [comment]
Agree, the most common terms would be for global horizontal insolation and plane-of-array insolation.

**Comment by toddkarin** [comment]
How about a suffix, e.g. `poa_global_int`, `poa_global_integrated` or `poa_global_interval`?
